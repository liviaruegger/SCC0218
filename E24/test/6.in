63
122142041 124668271
122143750 124673711
122795122 125426488
123480605 126178306
126171624 129097178
129523583 132541779
137517480 140692169
144091414 147305040
144231962 147445396
198496815 201250960
205365609 208025916
245972120 247500583
246204886 247724256
246284212 247800411
247079283 248553591
248478022 249839434
248483356 249844086
248487606 249845415
248508010 249840121
248524797 249833844
248984747 249447063
249515769 248954322
252712377 245816134
281263954 217493453
310091930 188667446
352727627 145849205
366292786 131974925
368106426 130118412
368179484 130043005
368320228 129896890
372580765 125447296
372676950 125341271
373260393 124693630
373395680 124536527
373407967 124509132
373407072 124504529
373399687 124486731
373383484 124456849
373143590 124198429
372844816 123880404
372805513 123839587
336708883 87737701
306938421 58065140
250412824 1916202
249386202 906962
249358364 880791
248435674 26350
248403581 0
248396843 2985
248389117 7021
248324288 53615
248297830 73774
248076242 276325
247795971 536037
247760531 569865
247729331 600389
162747998 84060060
147604646 98971735
147333073 99240391
138182832 108301350
127449277 119011324
127430362 119030676
123341642 123302414
