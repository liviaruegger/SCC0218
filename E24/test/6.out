63214584945143469
